src/core/lib/channel/channelz_registry.cc

```diff
@@ -29,39 +32,84 @@
 #include <grpc/support/log.h>
 #include <grpc/support/sync.h>
 
-#include <cstring>
-
 namespace grpc_core {
 namespace channelz {
+
 namespace {
+const int kPaginationLimit = 100;
+const grpc_millis kSweepIntervalMs = 5000;
```

Comment:
not related to this patch, but under memory pressure we probably would want to force GC. Would you consider adding a TODO here for that?

Reply:
Good idea.  Added a TODO.

Reply:
Thanks!